font-patcher

```diff
@@ -893,18 +987,9 @@ def get_glyph_dimensions(glyph):
         'height': bbox[3] + (-bbox[1]),
     }
 
-
-def use_scale_glyph(unicode_value, glyph_list):
-    """ Determines whether or not to use scaled glyphs for glyphs in passed glyph_list """
-    for i in glyph_list:
-        if isinstance(i, tuple):
-            if unicode_value >= i[0] and unicode_value <= i[1]:
-                return True
-        else:
-            if unicode_value == i:
-                return True
-    return False
-
+def get_glyph_dimensions(glyph):
```

Comment:
nice

Reply:
Code Climate would have complained anyhow ;-) because of code repetition.

(Well, I guess it is `off` for `font-patcher`, but it would have complained.)